core/static/js/dit.class.modal.js

```diff
@@ -71,28 +72,32 @@
     $button.addClass(CSS_CLASS_CLOSE_BUTTON);
     return $button;
   }
-  
+
   Modal.createContent = function() {
     var $content = $(document.createElement("div"));
     $content.addClass(CSS_CLASS_CONTENT);
     return $content;
   }
 
+  Modal.findFirstLink = function($container) {
```

Comment:
This selector is quite clearly targeting something specific to the Language Selector, so doesn't belong here. If this function is required, there should be a generic selector, e.g. `$("a", this)` (where `this` is a reference to the current modal node). An overwriting function, with a selector that is specific to the Language Selector should go in the LanguageSelectorDialog (or whatever it is called) class.

Of course, if the whole thing is specific to a Language Selector dialog and not a generic modal (e.g. something that also drives the Video modal), then perhaps the whole findFirstLink custom handling should just be kept out of the Modal class and added directly to the inheriting LanguageSelectorDialog class.